src/vsr/superblock.zig

```diff
@@ -716,6 +716,7 @@ pub fn SuperBlockType(comptime Storage: type) type {
             if (superblock.free_set.highest_address_acquired()) |address| {
                 staging.size += address * config.block_size;
             }
+            assert(staging.size <= staging.size_max);
```

Comment:
Should we also relate `staging.size` to `data_file_size_min` like we do below for `writing.size`?